.github/workflows/release.yml

```
@@ -19,37 +19,25 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)